shared/router-v2/header/index.desktop.tsx

```diff
@@ -180,7 +180,16 @@ class Header extends React.PureComponent<Props> {
               />
             )}
             <Kb.Box2 direction="horizontal" style={styles.topRightContainer}>
-              {flags.kbfsOfflineMode && <SyncingFolders />}
+              {flags.kbfsOfflineMode && (
+                <SyncingFolders
+                  negative={
+                    this.props.style &&
+                    this.props.style.backgroundColor &&
+                    this.props.style.backgroundColor !== Styles.globalColors.transparent &&
+                    this.props.style.backgroundColor !== Styles.globalColors.white
```

Comment:
@buoyad I'm really guessing here as to how to check the background color of the header. Would this work?
Keep in mind that the header props are not typed :(

Reply:
Yes this is alright. I don't think it would work on native though.